### PR TITLE
Webpack attempt

### PR DESCRIPTION
**Pull request by alschmiedt**
This is first attempt for just creating a bundle using webpack.

### CODE REVIEW

File: blockly-ws-search/src/css.js [hunk 1]
```diff
@@ -101,3 +103,5 @@ Blockly.Css.register([
     'display: flex;',
   '}',
 ]);
+
+console.log("here");
```
Comment (moniika) [inline blockly-ws-search/src/css.js]: remove debuggin line.

File: blockly-ws-search/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+module.exports = {
+    target: 'web',
+    mode: 'development',
+    entry: {
+        bundle: './src/WorkspaceSearch.js'
+    },
+    output: {
+        path: path.resolve(__dirname, 'build'),
+        filename: '[name].js',
+        // library: 'WorkspaceSearch',
```
Comment (moniika) [inline blockly-ws-search/webpack.config.js]: Add a TODO note about this so we don't forget what this is about.

File: blockly-ws-search/example/index.html [hunk 3]
```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>Workspace Search Playground</title>
   <script src="../node_modules/blockly/blockly.min.js"></script>
+  <script src="../build/bundle.js"></script>
```
Comment (rachel-fenichel) [inline blockly-ws-search/example/index.html]: Do you still need to import `main.js` as well as `../build/bundle.js`?

Comment (rachel-fenichel) [reply blockly-ws-search/example/index.html]: Never mind, I see now.

**Review by rachel-fenichel — APPROVED** [review]

File: blockly-ws-search/package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "Implementation of workspace search for Blockly.",
   "main": "src/workspace_search.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack",
+    "start": "http-server"
```
Comment (samelhusseini) [inline blockly-ws-search/package.json:8]: You can use webpack-dev-server here.

File: blockly-ws-search/src/css.js [hunk 5]
```diff
@@ -1,25 +1,16 @@
...
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 /**
  * @fileoverview Styling for workspace search.
  * @author kozbial@google.com (Monica Kozbial)
  */
 
+import * as Blockly from 'blockly';
+
```
Comment (samelhusseini) [inline blockly-ws-search/src/css.js:13]: Import only 'blockly/core' 
As a library unless you need access to Blocks and the JavaScript generator, it's best to only include core.

File: blockly-ws-search/src/WorkspaceSearch.js [hunk 6]
```diff
@@ -1,28 +1,19 @@
...
 /**
  * @fileoverview Object responsible for workspace search.
  * @author aschmiedt@google.com (Abby Schmiedt)
  */
 
-import './css.js'
 
-export default class WorkspaceSearch {
+import './css.js';
+import * as Blockly from 'blockly';
```
Comment (samelhusseini) [inline blockly-ws-search/src/WorkspaceSearch.js:14]: Same here, only import 'blockly/core'

**Pull request merged**